src/Infrastructure/Form/Regulation/LocationNewFormType.php

```diff
@@ -0,0 +1,146 @@
+                    'label_attr' => [
+                        'class' => 'required',
+                    ],
+                ],
+            )
+            ->add(
+                'cityCode',
+                HiddenType::class,
+            )
+            ->add(
+                'cityLabel',
+                TextType::class,
+                options: [
+                    'label' => 'regulation.location.city',
+                    'label_attr' => [
+                        'class' => 'required',
+                    ],
+                ],
+            )
+            ->add(
+                'roadName',
+                TextType::class,
+                options: [
+                    'label' => 'regulation.location.roadName',
+                    'help' => 'regulation.location.roadName.help',
+                    'required' => false,
+                ],
+            )
+            ->add(
+                'fromHouseNumber',
+                TextType::class,
+                options: [
+                    'required' => false,
+                    'label' => 'regulation.location.from_house_number',
+                ],
+            )
+            ->add(
+                'toHouseNumber',
+                TextType::class,
+                options: [
+                    'required' => false,
+                    'label' => 'regulation.location.to_house_number',
+                ],
+            );
+    }
+
+    private function getRoadTypeOptions(): array
+    {
+        $choices = [];
+
+        foreach (RoadTypeEnum::cases() as $case) {
+            $choices[sprintf('regulation.location.road.type.%s', $case->value)] = $case->value;
+        }
+
+        return [
+            'choices' => array_merge(
+                ['regulation.location.type.placeholder' => ''],
+                $choices,
+            ),
+            'label' => 'regulation.location.type',
+            'label_attr' => [
+                'class' => 'required',
+            ],
+        ];
+    }
+
+    private function getAdministratorOptions(array $administrators): array
+    {
+        $choices = [];
+
+        foreach ($administrators as $value) {
+            $choices[$value] = $value;
+        }
+
+        return [
+            'label' => 'regulation.location.administrator',
+            'label_attr' => [
+                'class' => 'required',
+            ],
+            'required' => false, // Due to error "An invalid form control with name='x' is not focusable"
+            'help' => 'regulation.location.administrator.help',
+            'choices' => array_merge(
+                ['regulation.location.administrator.placeholder' => ''],
+                $choices,
+            ),
+        ];
+    }
+
+    public function configureOptions(OptionsResolver $resolver): void
+    {
+        $resolver->setDefaults([
+            'validation_groups' => ['Default', 'html_form'],
```

Comment:
Sauf erreur il n'y a plus de validation spécifique à `html_form` dans `SaveLocationNew`

```suggestion
            'validation_groups' => ['Default'],
```

Par contre il y en a une dans `SaveMeasure` donc dans `MeasureFormType` c'est bon on peut le garder